integrationtests/gitjob/controller/controller_test.go

```diff
@@ -43,6 +44,9 @@ var _ = Describe("GitJob controller", func() {
 		)
 
 		JustBeforeEach(func() {
+			config.Set(&config.Config{
```

Comment:
This will affect all integration tests running after this one, I think? 
But that doesn't matter I guess, is this just for initialization? Instead of initializing the config in some tests, could we make the code work with an empty config instead?

Reply:
I've added the initialization to the suite instead of the tests where possible and initialized it empty where possible.